.github/workflows/ci_testing.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     env:
       SPICEQL_DEV_DB: True
       SPICEQL_LOG_LEVEL: DEBUG
+      SPICEQL_FASTAPI_LOG_LEVEL: DEBUG
```

Comment:
I think it's fine to share the same variable, just to not have more variables. I meant just to change the printout so it's different in the logs.